composer.json

```diff
@@ -22,17 +22,24 @@
     },
     "require-dev": {
         "composer/installers": "^1.9",
-        "johnpbloch/wordpress": "^5.4",
+        "johnpbloch/wordpress": "^6.1",
         "wp-graphql/wp-graphql": "^1.1.8",
         "squizlabs/php_codesniffer": "^3.5",
         "automattic/vipwpcs": "^2.3",
-        "wp-coding-standards/wpcs": "^2.3"
+        "wp-coding-standards/wpcs": "^2.3",
+        "wp-phpunit/wp-phpunit": "^6.3",
+        "yoast/phpunit-polyfills": "^2.0",
+        "phpunit/phpunit": "^9.6",
+        "codeception/module-asserts": "*",
+        "codeception/util-universalframework": "*",
+        "codeception/module-rest": "*",
+        "lucatume/wp-browser": "^3.1"
     },
     "scripts": {
         "cli": "docker-compose run --rm --workdir=/var/www/html/wp-content/plugins/wp-graphql-testcase --user $(id -u) wordpress wait_for_it $TEST_DB -s -t 300 --",
         "codeception": "codecept run wpunit --",
         "phpunit": "phpunit --",
-        "run-codeception": "env TEST_DB=mysql:3306 composer cli vendor/bin/codecept run wpunit",
+        "run-codeception": "env TEST_DB=mysql_phpunit:3306 composer cli vendor/bin/codecept run wpunit",
```

Comment:
This may be wrong, but I'm not sure why codeception used `mysql` while phpunit used `mysql_phpunit` before.. so I updated for both of them to use the same test DB.

Reply:
Because the DB isn't guarantee to be cleaned after the phpunit test, especially if it ends in a fail, so it's safer to use separate DBs